Wikipedia/Localizations/fr.lproj/Localizable.strings

```diff
@@ -734,19 +734,28 @@
 "table-of-contents-close-accessibility-label" = "Fermer la Table des matières";
 "table-of-contents-heading" = "Contenus";
 "table-title-other" = "Plus d’information";
+"talk-page-add-discussion-accessibility-label" = "Ajouter une discussion";
+"talk-page-discussion-accessibility-hint" = "Taper deux fois pour ouvrir un fil de discussion";
+"talk-page-discussion-read-accessibility-label" = "Lire";
+"talk-page-discussion-read-ipa-accessibility-attribute" = "rɛd";
```

Comment:
Concerns me a bit that these were left as the English pronunciation...

Reply:
I've updated French & Brazilian Portuguese translations upstream but I'm not sure about zh-hant yet. Maybe it'd help if we updated the description to provide more context

Reply:
I've added a support note for the zh-hant translations https://translatewiki.net/w/i.php?title=Support&lqt_method=talkpage_new_thread&lqt_subject_field=About+%5B%5BWikimedia%3AWikipedia-ios-talk-page-discussion-read-ipa-accessibility-attribute%2Fzh-hant%5D%5D#About_.5B.5BWikimedia:Wikipedia-ios-talk-page-discussion-read-ipa-accessibility-attribute.2Fzh-hant.5D.5D_and_.5B.5BWikimedia:Wikipedia-ios-talk-page-discussion-unread-ipa-accessibility-attribute.2Fzh-hant.5D.5D_56532

Reply:
@nambatee awesome, thanks!